NOTICE

```diff
@@ -0,0 +1,10 @@
+Parse Benchmark
+
+Copyright 2019-present Parse Platform
+
+This product includes software developed at Parse Platform.
+www.parseplatform.org
+
+---
+
+As of August 19, 2019, the original author Diamond Lewis has transferred this code to the Parse Platform organization.
```

Comment:
Do you have a reference for this transfer?

Reply:
The [first commit](https://github.com/parse-community/benchmark/commit/b5f23b8742e5201b3eae6cbfd350a2b36472055b) appears to be a transfer by Diamond, I'm not sure though

Reply:
Yes, makes sense to assume that, at least he seems to have written the whole thing by his own before transferring it.